tests/unit_tests/data_steward/tools/generate_ext_tables_test.py

```diff
@@ -159,7 +159,7 @@ def test_generate_ext_table_queries(self, mock_mapping_tables,
                     common.PERSON, common.DEATH, common.FACT_RELATIONSHIP
```

Comment:
Line 63 in this test could be changed to:
```
        ext_table = f'foo{gen_ext.EXT_TABLE_SUFFIX}'
```

Line 101 could be changed to:
```
        mapping_observation = f'{gen_ext.MAPPING_PREFIX}{observation_table_id}'
```

Line 140-141 could become:
```
        hpo_bq_list.append(f'("{gen_ext.RDR}", "{gen_ext.PPI_PM}")')
```